pkg/system/filesys_windows.go

```diff
@@ -0,0 +1,62 @@
+// +build windows
+
+package system
+
+import (
+	"os"
+	"regexp"
+	"syscall"
+)
+
+// Volume Path aware implementation of MkdirAll for Windows.
+func MkdirAll(path string, perm os.FileMode) error {
+	re := regexp.MustCompile(`^\\\\\?\\Volume{[a-z0-9-]+}$`)
+	if re.MatchString(path) {
+		return nil
+	}
+
+	// Fast path: if we can tell whether path is a directory or file, stop with success or error.
+	dir, err := os.Stat(path)
+	if err == nil {
+		if dir.IsDir() {
+			return nil
+		}
+		return &os.PathError{
+			Op:   "mkdir",
+			Path: path,
+			Err:  syscall.ENOTDIR,
+		}
+	}
+
+	// Slow path: make sure parent exists and then call Mkdir for path.
+	i := len(path)
+	for i > 0 && os.IsPathSeparator(path[i-1]) { // Skip trailing path separator.
+		i--
+	}
+
+	j := i
+	for j > 0 && !os.IsPathSeparator(path[j-1]) { // Scan backward over element.
+		j--
+	}
+
+	if j > 1 {
+		// Create parent
+		err = MkdirAll(path[0:j-1], perm)
+		if err != nil {
+			return err
+		}
+	}
+
+	// Parent now exists; invoke Mkdir and use its result.
+	err = os.Mkdir(path, perm)
+	if err != nil {
```

Comment:
This implementation is purposefully identical to os.MkdirAll, other than the added regexp at the top.  We should keep the rest of the function as-is in order to maintain that.